test/TensorFlow/crashers.swift

```diff
@@ -353,3 +353,17 @@ public func constFoldingBug() {
     return
   }
 }
+
+public func SR8419(iterationCount: Int) {
```

Comment:
Could you please add a comment before this function explaining the initial crash?
Sth like: `// This function failed SIL verification due to a non-inlined SILDebugLocation.`
Thanks!

Reply:
Fixed